pkg/addon/addon_update_test.go

```diff
@@ -109,17 +70,13 @@ func TestUpdateAddonStatusLifecycle(t *testing.T) {
 			Resources: []addonmgrv1alpha1.ObjectStatus{},
 		},
 	}
-	err = updater.client.Create(ctx, testAddon, &client.CreateOptions{})
-	Expect(err).To(BeNil())
+	err := updater.client.Create(ctx, testAddon, &client.CreateOptions{})
+	g.Expect(err).ToNot(gomega.HaveOccurred())
 
-	existingAddon, err := updater.getExistingAddon(ctx, fmt.Sprintf("%s/%s", testNamespace, testAddonName))
-	Expect(err).To(BeNil())
-	Expect(existingAddon).NotTo(BeNil())
+	existingAddon, err := updater.getExistingAddon(ctx, testNamespace, testAddonName)
+	g.Expect(err).ToNot(gomega.HaveOccurred())
+	g.Expect(existingAddon).ToNot(gomega.BeNil())
 
-	err = updater.UpdateAddonStatusLifecycle(ctx, testNamespace, testAddonName, "install", "Succeeded")
-	if err != nil {
-		fmt.Printf(" update addon status err %#v", err)
-	}
-	Expect(err).To(BeNil())
-	cleanup(testEnv)
+	err = updater.UpdateAddonStatusLifecycle(ctx, testNamespace, testAddonName, addonmgrv1alpha1.Install, addonmgrv1alpha1.Succeeded)
+	g.Expect(err).ToNot(gomega.HaveOccurred())
```

Comment:
Are there any status assertions we should be doing in this test? I do see from before though that none were being done.

Reply:
This isn't actually an e2e test though, so less important I suppose.

Reply:
We have sort of e2e testing using `testEnv` [here](https://github.com/keikoproj/addon-manager/blob/master/controllers/addon_controller_test.go#L43)

Reply:
BDDs we have [here](https://github.com/keikoproj/addon-manager/blob/master/test-bdd/main_test.go) but we should revisit those as they're not updated.